Distribution/Data/teleporters.json

```diff
@@ -40,9 +40,9 @@
     "back": true
   },
   {
-    "src": { "map": "Felucca", "loc": [589, 1637, 0] },
-    "dst": { "map": "Felucca", "loc": [6575, 888, 0] },
-    "back": false
+    "src": { "map": "Felucca", "loc": [588, 1637, 0] },
```

Comment:
Let's make a separate PR for the teleporter updates so we can merge those quickly.